templates/components/products/card.html

```diff
@@ -29,18 +29,40 @@
     {{/if}}>
     <figure class="card-figure">
         {{#if stock_level '===' 0}}
-            {{#if theme_settings.pdp_sold_out_label '===' ''}}
-                {{> components/products/product-badge
-                    badge-type='sold-out'
-                    badge_view=theme_settings.product_sold_out_badges
-                    badge_label=(lang "page_builder.pdp_sold_out_label")
-                }}
-            {{else}}
-                {{> components/products/product-badge
-                    badge-type='sold-out'
-                    badge_view=theme_settings.product_sold_out_badges
-                    badge_label=theme_settings.pdp_sold_out_label
-                }}
+            {{#if show_cart_action}}
+                {{#if theme_settings.pdp_sold_out_label '===' ''}}
+                    {{> components/products/product-badge
+                        badge-type='sold-out'
+                        badge_view=theme_settings.product_sold_out_badges
+                        badge_label=(lang "page_builder.pdp_sold_out_label")
+                    }}
+                {{else}}
+                    {{> components/products/product-badge
+                        badge-type='sold-out'
+                        badge_view=theme_settings.product_sold_out_badges
+                        badge_label=theme_settings.pdp_sold_out_label
+                    }}
+                {{/if}}
+            {{/if}}
+        {{else if has_options '===' false}}
+            {{#if stock_level '===' null}}
+                {{#if show_cart_action}}
```

Comment:
@bc-alexsaiannyi could we combine first two conditions (line 48 and 49) via {{#and}}?

Reply:
@probably yes, but it will look like:
`{{#and (if stock_level '===' null) show_cart_action }}`

Do you think it looks better?

Reply:
@bc-alexsaiannyi not sure, but maybe like this `{{#and (unless stock_level) show_cart_action }}`